apps/antalmanac/src/components/RightPane/CoursePane/CoursePaneButtonRow.tsx

```diff
@@ -12,7 +12,8 @@ import {
     type SxProps,
 } from '@mui/material';
 import { ArrowBack, Visibility, Refresh } from '@mui/icons-material';
-import RightPaneStore, { type SectionTableColumn } from '../RightPaneStore';
+import RightPaneStore from '../RightPaneStore';
+import useColumnStore, { SECTION_TABLE_COLUMNS, type SectionTableColumn } from '$stores/ColumnStore';
```

Comment:
I prefer to use named exports because there's slightly better LSP support for finding them.

```suggestion
import { useColumnStore, SECTION_TABLE_COLUMNS, type SectionTableColumn } from '$stores/ColumnStore';
```